packages/webpack/src/config/base.js

```diff
@@ -55,21 +54,29 @@ export default class WebpackBaseConfig {
     return this.buildContext.buildOptions.loaders
   }
 
-  normalizeTranspile () {
-    // include SFCs in node_modules
-    const items = [/\.vue\.js/i]
+  get modulesToTranspile () {
+    return [
+      /\.vue\.js/i, // include SFCs in node_modules
```

Comment:
```suggestion
      /\.vue\.js/i, // include SFCs in node_modules
```

Shouln't we add `.vue` extensions as well?

Reply:
vue won’t be here since it has been transformed by vue-loader